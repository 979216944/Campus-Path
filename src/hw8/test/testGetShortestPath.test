# Verify that the program correctly gets shortest path 
# between two buildings on campus map

# find shortest path from UBS to CSE
r
UBS
CSE

# find shortest path from MOR to CSE
r
MOR
CSE

# find shortest path from CSE to UBS
r
CSE
UBS

# find shortest path from CSE to MOR
r
CSE
MOR

# find shortest path from CSE to FSH
r
CSE
FSH

# quit
q