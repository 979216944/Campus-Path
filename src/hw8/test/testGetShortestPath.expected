Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Verify that the program correctly gets shortest path
# between two buildings on campus map

# find shortest path from UBS to CSE
Abbreviated name of starting building: Abbreviated name of ending building: Path from University Bookstore to Paul G. Allen Center for Computer Science & Engineering:
Walk 136 feet S to (1374, 622)
Walk 46 feet S to (1371, 643)
Walk 44 feet S to (1371, 664)
Walk 300 feet S to (1371, 807)
Walk 148 feet S to (1371, 878)
Walk 99 feet E to (1418, 880)
Walk 23 feet E to (1430, 881)
Walk 104 feet E to (1480, 881)
Walk 62 feet E to (1510, 877)
Walk 22 feet E to (1522, 878)
Walk 28 feet SE to (1528, 890)
Walk 27 feet SE to (1537, 900)
Walk 330 feet SE to (1669, 988)
Walk 36 feet SE to (1684, 995)
Walk 100 feet E to (1730, 1008)
Walk 77 feet SE to (1754, 1034)
Walk 35 feet S to (1749, 1049)
Walk 75 feet E to (1786, 1049)
Walk 38 feet SE to (1798, 1062)
Walk 33 feet S to (1801, 1077)
Walk 124 feet SE to (1846, 1117)
Walk 73 feet SE to (1872, 1141)
Walk 49 feet SE to (1890, 1159)
Walk 117 feet SE to (1930, 1198)
Walk 220 feet SE to (2003, 1276)
Walk 30 feet SE to (2013, 1284)
Walk 96 feet SE to (2054, 1303)
Walk 150 feet E to (2127, 1311)
Walk 77 feet S to (2129, 1347)
Walk 194 feet SE to (2166, 1434)
Walk 114 feet E to (2221, 1432)
Walk 61 feet S to (2225, 1461)
Walk 42 feet SE to (2234, 1480)
Walk 50 feet SE to (2255, 1492)
Walk 29 feet S to (2257, 1505)
Walk 172 feet SE to (2307, 1571)
Walk 116 feet S to (2302, 1627)
Walk 141 feet S to (2305, 1694)
Walk 25 feet SW to (2295, 1701)
Walk 68 feet W to (2261, 1707)
Walk 18 feet S to (2260, 1716)
Total distance: 3731 feet

Enter an option ('m' to see the menu):
# find shortest path from MOR to CSE
Abbreviated name of starting building: Abbreviated name of ending building: Path from Moore Hall to Paul G. Allen Center for Computer Science & Engineering:
Walk 20 feet NE to (2322, 1853)
Walk 95 feet NW to (2287, 1826)
Walk 40 feet NW to (2270, 1816)
Walk 80 feet NE to (2287, 1782)
Walk 79 feet N to (2298, 1745)
Walk 52 feet N to (2291, 1721)
Walk 62 feet W to (2260, 1716)
Total distance: 426 feet

Enter an option ('m' to see the menu):
# find shortest path from CSE to UBS
Abbreviated name of starting building: Abbreviated name of ending building: Path from Paul G. Allen Center for Computer Science & Engineering to University Bookstore:
Walk 18 feet N to (2261, 1707)
Walk 68 feet E to (2295, 1701)
Walk 25 feet NE to (2305, 1694)
Walk 141 feet N to (2302, 1627)
Walk 116 feet N to (2307, 1571)
Walk 172 feet NW to (2257, 1505)
Walk 29 feet N to (2255, 1492)
Walk 50 feet NW to (2234, 1480)
Walk 42 feet NW to (2225, 1461)
Walk 61 feet N to (2221, 1432)
Walk 114 feet W to (2166, 1434)
Walk 194 feet NW to (2129, 1347)
Walk 77 feet N to (2127, 1311)
Walk 150 feet W to (2054, 1303)
Walk 96 feet NW to (2013, 1284)
Walk 30 feet NW to (2003, 1276)
Walk 220 feet NW to (1930, 1198)
Walk 117 feet NW to (1890, 1159)
Walk 49 feet NW to (1872, 1141)
Walk 73 feet NW to (1846, 1117)
Walk 124 feet NW to (1801, 1077)
Walk 33 feet N to (1798, 1062)
Walk 38 feet NW to (1786, 1049)
Walk 75 feet W to (1749, 1049)
Walk 35 feet N to (1754, 1034)
Walk 77 feet NW to (1730, 1008)
Walk 100 feet W to (1684, 995)
Walk 36 feet NW to (1669, 988)
Walk 330 feet NW to (1537, 900)
Walk 27 feet NW to (1528, 890)
Walk 28 feet NW to (1522, 878)
Walk 22 feet W to (1510, 877)
Walk 62 feet W to (1480, 881)
Walk 104 feet W to (1430, 881)
Walk 23 feet W to (1418, 880)
Walk 99 feet W to (1371, 878)
Walk 148 feet N to (1371, 807)
Walk 300 feet N to (1371, 664)
Walk 44 feet N to (1371, 643)
Walk 46 feet N to (1374, 622)
Walk 136 feet N to (1374, 557)
Total distance: 3731 feet

Enter an option ('m' to see the menu):
# find shortest path from CSE to MOR
Abbreviated name of starting building: Abbreviated name of ending building: Path from Paul G. Allen Center for Computer Science & Engineering to Moore Hall:
Walk 62 feet E to (2291, 1721)
Walk 52 feet S to (2298, 1745)
Walk 79 feet S to (2287, 1782)
Walk 80 feet SW to (2270, 1816)
Walk 40 feet SE to (2287, 1826)
Walk 95 feet SE to (2322, 1853)
Walk 20 feet SW to (2317, 1860)
Total distance: 426 feet

Enter an option ('m' to see the menu):
# find shortest path from CSE to FSH
Abbreviated name of starting building: Abbreviated name of ending building: Path from Paul G. Allen Center for Computer Science & Engineering to Fishery Sciences Building:
Walk 18 feet N to (2261, 1707)
Walk 90 feet W to (2218, 1696)
Walk 134 feet W to (2157, 1675)
Walk 200 feet SW to (2067, 1714)
Walk 21 feet W to (2055, 1718)
Walk 134 feet SW to (1996, 1743)
Walk 103 feet SW to (1952, 1766)
Walk 151 feet SW to (1889, 1797)
Walk 84 feet SW to (1852, 1813)
Walk 54 feet W to (1828, 1822)
Walk 48 feet SW to (1815, 1841)
Walk 57 feet W to (1788, 1844)
Walk 38 feet SW to (1771, 1851)
Walk 35 feet NW to (1764, 1836)
Walk 20 feet W to (1755, 1839)
Walk 55 feet SW to (1736, 1858)
Walk 17 feet SE to (1742, 1863)
Walk 27 feet SW to (1732, 1871)
Walk 32 feet NW to (1721, 1860)
Walk 171 feet SW to (1671, 1924)
Walk 45 feet SW to (1652, 1935)
Walk 38 feet NW to (1637, 1925)
Walk 69 feet NW to (1610, 1907)
Walk 87 feet NW to (1575, 1885)
Walk 98 feet NW to (1536, 1858)
Walk 98 feet NW to (1498, 1830)
Walk 64 feet NW to (1474, 1814)
Walk 80 feet NW to (1446, 1788)
Walk 37 feet W to (1432, 1789)
Walk 198 feet NW to (1356, 1732)
Walk 82 feet SW to (1328, 1761)
Walk 25 feet NW to (1319, 1756)
Walk 83 feet W to (1282, 1743)
Walk 102 feet W to (1234, 1736)
Walk 33 feet W to (1218, 1736)
Walk 61 feet W to (1189, 1730)
Walk 15 feet NW to (1183, 1727)
Walk 93 feet W to (1138, 1725)
Walk 150 feet W to (1066, 1722)
Walk 27 feet S to (1066, 1735)
Walk 35 feet SE to (1079, 1744)
Walk 81 feet SW to (1062, 1780)
Total distance: 3092 feet

Enter an option ('m' to see the menu):
# quit
